packages/core/CalendarManager.ts

```diff
@@ -21,7 +21,7 @@ import getCalendarsEvents from "./getCalendarsEvents";
 const log = logger.getChildLogger({ prefix: ["CalendarManager"] });
 
 export const getCalendarCredentials = (credentials: Array<CredentialPayload>) => {
-  const calendarCredentials = getApps(credentials)
+  const calendarCredentials = getApps(credentials, true)
```

Comment:
This function is used for connected calendars, so we can improve performance by only going through an array of installed app metadata.